ci/openshift.yml

```diff
@@ -66,7 +67,42 @@ jobs:
     runs-on: ubuntu-20.04
     environment: production
 
+    outputs:
+        ROUTE: ${{ steps.deploy-and-expose.outputs.route }}
+        SELECTOR: ${{ steps.deploy-and-expose.outputs.selector }}
+
     steps:
+    - name: Check if secrets exists
```

Comment:
I like the DX this produces even if it makes the workflow a bit longer. From what I understand, the tight coupling to standard secret names is to pave the path for potential future enhancements like setting the secrets programmatically?

Reply:
Yes - we are trying to 'standardize' the secret names to use. Eg, these ones match https://github.com/redhat-actions/oc-login#example-workflow-step.

Do you see a way to do it with more flexibility in the secret names?

re: workflow length, We hope to capture this into an action though it's a little more complex because you'd need an access token w/ read secrets permission.

Reply:
> Do you see a way to do it with more flexibility in the secret names?

Not offhand but will think about it.